tools/build_go_exe.sh

```diff
@@ -89,7 +89,7 @@ cd $GOPATH_EXE_DIR
 # previous native build. if the sha1 matches, then we can skip the rebuild.
 NATIVE_FILENAME=".native-${EXE_BASE_NAME}"
 NATIVE_SHA1SUM_FILENAME="${NATIVE_FILENAME}.sha1sum"
-go build -o $NATIVE_FILENAME
+go build -a -ldflags "${GO_LDFLAGS}" -o $NATIVE_FILENAME
```

Comment:
What about the case where `GO_LDFLAGS` is not defined?

Reply:
Is the SHA time-dependent, or does the addition of the `-a` flag now mean that we will always build the platform-specific versions below?

Reply:
> What about the case where GO_LDFLAGS is not defined?

An empty string will be used instead. We are not using bash strict mode (i.e. '-u' option) in this script. None the less I have added variable initialization.

> Is the SHA time-dependent, or does the addition of the -a flag now mean that we will always build the platform-specific versions below?

Neither. We are just making sure that before skipping the rebuild for other platforms, **nothing** has changed. We achieve it by forcefully rebuilding both the CLI and its dependencies. Basing on my quick tests, SHA is not time-dependent.